### ISSUE TITLE
NodeJS Version not updated after changed it!

### ISSUE THREAD

**Issue by Alamnzr123** [leokhoa/laragon#763]
![Screenshot (10255)](https://github.com/leokhoa/laragon/assets/54619997/a40daee8-828c-45d8-9795-efe74f505fc0)

Hey all, i have issues nodeJS version not update, i already installed new version Laragon 6.0 and try to restart the command but still same

**Comment by ericsizemore** [comment]
When you say `try to restart the command` do you mean you already tried closing then reopening the Terminal? I had this same issue but I seem to remember just closing and opening the Terminal again and it worked fine.

**Comment by Alamnzr123** [comment]
> When you say `try to restart the command` do you mean you already tried closing then reopening the Terminal? I had this same issue but I seem to remember just closing and opening the Terminal again and it worked fine.

Yes, still same after i closing then reopening the terminal

**Comment by khsuzan** [comment]
Sometimes restart the machine do the hack. Have you wanna try

**Comment by alxndr-w** [comment]
I don't think this is related to Laragon itself. Refering to: <https://stackoverflow.com/questions/77517264/switching-node-version-in-windows>

esp.: https://stackoverflow.com/a/77517327

> Close terminal and reopen again. When you update nvm version, for some reason running node -v in same terminal returns old result on windows. Closing and reopening terminal helps for me.
If reopening terminal does not help, then you may have 2 different installations. node command you run points to something different from the nvm. You can check system environment variables in Windows and see if in path you have 2 different NodeJS installation routes provided. If yes only leave one, so that both nvm and node point to same installation.
